em-annotation/requirements.yaml

```diff
@@ -1,19 +1,19 @@
 dependencies:
   - name: em-npa
     version: ~0.0.9
-    repository: '@hmcts'
+    repository: '@hmctspublic'
```

Comment:
Have you checked if all these charts are latest and available in hmctspublic?

Reply:
Yes. Checked now. These are available on hmctspublic